pkgs/development/python-modules/django-auth-ldap/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, ldap , django_2_2 
+, mock
+}:
+
+buildPythonPackage rec {
+  pname = "django-auth-ldap";
+  version = "1.7.0";
```

Comment:
out of date https://github.com/django-auth-ldap/django-auth-ldap/releases/tag/2.0.0

Reply:
fixed